### PR TITLE
@mate-academy/scripts package

### PR DESCRIPTION
**Pull request by YuriiHoliuk**

### CODE REVIEW

File: mate-scripts/configs/git-hooks/pre-commit [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
```
Comment (ivadimko) [inline mate-scripts/configs/git-hooks/pre-commit:1]: no more husky, only hardcore 🙂

File: mate-scripts/rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    ...Object.keys(pkg.dependencies || {}),
+    'path',
+    'fs',
+    'child_process',
+  ],
+  plugins: [
+    typescript({
+      typescript: require('typescript'),
+      tsconfig: './tsconfig.json',
+    }),
```
Comment (ivadimko) [inline mate-scripts/rollup.config.js:24]: wow, typescript support for src files, great!

File: mate-scripts/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+program
+  .command('init')
+  .description('init linters and configs')
+  .action(commandFactory.make(InitCommand));
+
+program
+  .command('start')
+  .description('run development server')
+  .action(commandFactory.make(StartCommand));
```
Comment (ivadimko) [inline mate-scripts/src/index.ts:31]: 😮 nice flow

File: mate-scripts/configs/git-hooks/pre-commit (hunk 1 above)
Comment (YuriiHoliuk) [reply mate-scripts/configs/git-hooks/pre-commit:1]: Just to avoid additional configs inside homework repos

**Pull request merged**